packages/suite/src/middlewares/wallet/coinjoinMiddleware.ts

```diff
@@ -55,6 +62,35 @@ export const coinjoinMiddleware =
             api.dispatch(coinjoinAccountActions.pauseCoinjoinSessionByDeviceId(action.payload.id));
         }
 
+        if (action.type === COINJOIN.CLIENT_SESSION_PHASE) {
+            const account = api
+                .getState()
+                .wallet.coinjoin.accounts.find(({ key }) => key === action.payload.accountKeys[0]); // all accounts asocciated with the accountKeys array have the same session phase
```

Comment:
scenario: 
- there are 2 accountKeys in the payload
- one of the account was removed just a second ago (by device disconnection), coinjoin package doesnt know that yet, therefore account is still registered, but account is no longer present in reducer
shouldnt it be like?:
```
const accounts = wallet.coinjoin.accounts.find(({ key }) => action.payload.includes(key)
```

Reply:
Great catch! Definitely more secure

[265103c](https://github.com/trezor/trezor-suite/pull/7057/commits/265103ca917bebd1e4ef16a1ad6ba00d78d03941)